test/check-avocado-selenium

```diff
@@ -56,12 +56,22 @@ done;
 
         self.allow_journal_messages("Error receiving data: Connection reset by peer",
                                     "Error sending data: Connection reset by peer",
-                                    "connection unexpectedly closed by peer"
+                                    "connection unexpectedly closed by peer",
+                                    ".*docker.sock.*No such file or directory",
+                                    "ahoj notice",
+                                    'Error: Could not obtain the password to be scored',
+                                    'Password quality check failed:',
+                                    'The password is a palindrome',
+                                    'The password is shorter than 8 characters',
+                                    '.*dev=.tmpfs..*scontext=system_u:system_r:syslogd_t:s0 tcontext=system_u:object_r:tmpfs_t:s0.*'
```

Comment:
it is good idea, because I can't imagine any useful journal messages, in case test passed and in case test fails, in journal file there are all messages, also ignored one.